test/extended/util/annotate/rules.go

```diff
@@ -88,6 +88,9 @@ var (
 			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(delayed binding\)\] topology should provision a volume and schedule a pod with AllowedTopologies`,
 			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(immediate binding\)\] topology should fail to schedule a pod which has topologies that conflict with AllowedTopologies`,
 			`\[sig-storage\] In-tree Volumes \[Driver: vsphere\] \[Testpattern: Dynamic PV \(immediate binding\)\] topology should provision a volume and schedule a pod with AllowedTopologies`,
+			// Skip openstack-specific storage tests in preparation for in-tree cinder provisioner removal
+			// coming with k8s 1.26
+			`\[sig-storage\].*\[Driver: cinder\]`,
```

Comment:
There should be a link to a bug to fix the tests.

Reply:
Done - https://issues.redhat.com/browse/OCPBUGS-5029.